rasa/model.py

```diff
@@ -515,3 +515,104 @@ async def update_model_with_new_domain(
     domain = await importer.get_domain()
 
     domain.persist(model_path / DEFAULT_DOMAIN_PATH)
+
+
+def get_models_for_finetuning(
+    previous_model_file: Optional[Union[Path, Text]]
+) -> Tuple[Optional["Agent"], Optional["Interpreter"]]:
```

Comment:
All the other similar functions in this module return paths rather than actual models, does it make sense to keep it consistent?

Reply:
Mhm, but then we'd load things repeatedly (e.g. the `interpreter` would be loaded for the core training and for NLU training). What I saw from testing it, is that loading the entire pipeline etc. takes quite a while, that's why I opted for doing it as early as possible.

Reply:
Will do this later 👍

Reply:
done